pkgs/racket-doc/scribblings/reference/contracts.scrbl

```diff
@@ -1944,6 +1944,22 @@ is bound to vectors of two elements, the exported identifier and a
 syntax object for the expression that produces the contract controlling
 the export.
 
+Example usage:
+
+@codeblock|{
+ #lang racket/base
+
+ (require racket/contract)
+
+ (provide
+  (contract-out
+   [recip (-> (and/c real? non-zero?)
+              (between/c -1 1))]))
+
+ (define (recip x) (/ 1 x))
+ (define (non-zero? x) (not (= x 0)))
```

Comment:
Perhaps `(zero? x)`?

Reply:
Or perhaps, using `not/c`:

````
(and/c real? (not/c zero?))
```

And then you can drop `(define (non-zero? x) ...)`.

Reply:
Thanks, good suggestion